ex3/Premier league.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Good.</p><p>Now please find a concise solution.</p><p>Tip: Create a boolean series with True where the home team wins, and then use <code>.mean()</code>	</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>